### ISSUE TITLE
Enhance build system info display with cpu and memory info

### ISSUE THREAD

**Issue by wsfulton** [travis-ci/travis-ci#5311]
Could you enhance your Build system information with the cpu and memory information?
I was thinking the output from the following should suffice:

lscpu && cat /proc/cpuinfo | grep "model name" && cat /proc/meminfo | grep MemTotal

**Comment by BanzaiMan** [comment]
I don't think that will work. The build system information is gathered at the image provisioning time, which is (usually) not the same as the VM used at run time.

**Comment by joshk** [comment]
Also, container based builds will report more cpus and ram as they don't see the cpu and ram limits they are given.
I do like this idea though, but it may have to wait until early next year.

**Comment by BanzaiMan** [comment]
I think this is a duplicate of #4068.
